### DISCUSSION TITLE
Properties view

### DISCUSSION THREAD

**Discussion by CodeDoctorDE** [LinwoodDev/Butterfly#241] [Ideas]
Elements are currently not very exact in the current version.
To fix this, I have a new idea: A properties view on the right side or an extra view in the mobile version in the current context menu (an option "Properties").
With this you can click on an element and see all properties (like position,...).
The current context menu won't be opened anymore with a simple click and will only have Cut, Copy, Paste and Delete.
Move and Duplicate will be added to this new view. The current context menu will get opened with a long click or a right click.
With this you can be able to select multiple elements at the same time and edit them.

**What do you think? Please comment on this discussion!**

Update:
It can look like this:
1.
![](https://cdn.discordapp.com/attachments/363912006857785347/1006147360499445801/unknown.png)
2. For example with the eraser painter
![](https://cdn.discordapp.com/attachments/363912006857785347/1006148597210959902/unknown.png)
3. Another idea for a look
![](https://cdn.discordapp.com/attachments/363912006857785347/1006150317450219530/unknown.png)

**Comment by Shahin-rmz** [comment]
I am in favor of minimalism. 
Less is better than more.
Small and neat is better than big and bulky.
3 is better imo.
